### DISCUSSION TITLE
Switch relay OFF, if Mosquitto broker (Home Assistant) not available

### DISCUSSION THREAD

**Discussion by andrewbpowers** [arendst/Tasmota#14734] [Q&A] [answered]
Hello!

I am using an ESP8266 and a relay with Tasmota on a Home Assistant server. So far everything went well. Now for the first time the following happened... the HA server was not reachable/switched off, but the relay was switched on. 

How can I achieve that as soon as the HA server/Mosquitto broker is not reachable, the relay switches off automatically?

I read somewhere something about "LWT" (SetOption10), but I am not sure if this is the right option or configuration.

I hope it was somehow understandable!?
;-)

Thanks for any help and best regards!

**Comment by barbudor** [comment, marked as answer]
LWT (Last Will and Testament) give the status of the Tasmota device.
At startup, Tasmota publish "Online" to state/%topic%/LWT and inform the broker that if the connection should be broken, th ebroker should automatically publish "Offline" on the same topic. This is to inform the infrastructure that the Tasmota device is gone abruptly.
So not what's you are looking for

What you are looking for is a Tasmota Rule based on trigger `ON Mqtt#Disconnect DO Power OFF ENDON`
Read the doc section about rules on how to use that.

**Reply by andrewbpowers** [reply]
First of all, thanks for info and the tip! :-)

I will try to create a rule. Maybe I have then still the one or other question about it...
;-)

**Reply by andrewbpowers** [reply]
I try this rule:

`rule2 on Mqtt#Disconnected do power1 off endon`

When I disconnect the network cable from the HA server, the relay still switches off for some time.
As it should be! 

Once again, thank you very much! :-)
